src/files/coveragefile.ts

```diff
@@ -4,6 +4,7 @@ export enum CoverageType {
     CLOVER,
     COBERTURA,
     JACOCO,
+    LLVM_COV_JSON, // LLVM coverage JSON format
```

Comment:
We probably don't need the comment here (I would also shorten the enum to just LLVM to match the others).


```suggestion
    LLVM,
```